app/templates/editions/octane.hbs

```diff
@@ -121,6 +95,8 @@
       Yes! Octane is stable and well supported.
       It is the present and future of Ember, so you are saving yourself some upgrade work down the line.
       Be sure to budget some time for learning and experimentation.
+      From Ember 3.15 onwards, if you start a new app and use Ember CLI commands like `ember generate component` to create
+      your files, you are using Octane!
```

Comment:
I wasn't sure if this answer belongs well to the question `I'm starting a new app at work. Should I use Octane?`.

From this answer, it seemed to me like that there might be people who are unsure if their app is already Octane. _How could I check if I am using Octane?_

Did I misunderstand your intent?

Reply:
I like the idea of making this be its own question. Thanks for the suggestion!